CONTRIBUTING.md

```diff
@@ -4,6 +4,7 @@ Folow a simple workflow:
 * Create Issue to explain what is wrong or missing
 * Fork this repository
 * Create a Pull Request referencing the issue you're adressing
+* enabled GitHub actions on your fork to automatically run generation steps
```

Comment:
is this something people need to do manually?

Reply:
Iirc it depends on the defaults you have set